src/App.css

```diff
@@ -0,0 +1,129 @@
+*{
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.container {
+  width: 550px;
+  min-width: 550px;
+  background: #23252C no-repeat center fixed;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  font-family: 'Inter', sans-serif;
+
+}
+
+.app {
+  margin-top: 45px;
+  margin-bottom: 45px;
+  background: #1A1B21;
+  border-radius: 10px 10px;
+}
+
+.section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #FFFFFF;
+  max-width: 320px; 
+  
+}
+
+.name {
+  margin-top: 20px;
+  font-weight: 700;
+  font-size: 25px;
+  text-align: center;
+}
+
+.dsgn {
+  font-weight: 400;
+  font-size: 13px;
+  text-align: center;
+  color: #F3BF99;
+  margin-top: 2px;
+  margin-bottom: 8px;  
+}
+
+.website {
+  font-weight: 400;
+  font-size: 10px;
+  text-align: center;
+  color: #F5F5F5;  
+  margin-bottom: 15px;
+}
+
+button {
+  background: #FFFFFF;
+  border: 1px solid #D1D5DB;
+  box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.05);
+  border-radius: 6px;
+  display: flex;
+  gap: 8px;
+  justify-content: center;
+  padding: 9px 11px;
+  max-width: 115px;
+  width: 115px;
```

Comment:
When you specify max-width with some value, don't specify width with same value. Otherwise, there will be no point of using max-width and width will be same 115px only even if you reduce the screen width. When you use max-width: 115px , make width: 100%. It means let width occupy 100% of maximum 115px width or any width value.